src/entities/DFGVariable.jl

```diff
@@ -55,3 +54,129 @@ DFGVariable(label::Symbol, softtype::T;
             solvable::Int=1,
             _internalId::Int64=0) where {T <: InferenceVariable} =
     DFGVariable{T}(label, now(), tags, estimateDict, solverDataDict, smallData, bigData, solvable, DFGNodeParams(solvable, _internalId))
+
+"""
+    $SIGNATURES
+DFGVariable constructors.
+"""
+function DFGVariable(label::Symbol, _internalId::Int64 = 0) #where {T <:InferenceVariable}
+    st = stacktrace()
+    @warn "DFGVariable(label::Symbol, _internalId::Int64 = 0) is deprecated please use DFGVariable(label::Symbol, softtype::T, _internalId::Int64 = 0) where T <: InferenceVariable. Enable DEBUG logging for the stack trace."
+    @debug st
+    T = InferenceVariable
+    DFGVariable(label, now(), Symbol[],
+                  Dict{Symbol, MeanMaxPPE}(),
+                  Dict{Symbol, VariableNodeData{T}}(:default => VariableNodeData()),
+                  Dict{String, String}(),
+                  Dict{Symbol,AbstractBigDataEntry}(), 0, _internalId)
+end
+DFGVariable(label::Symbol, softtype::T, _internalId::Int64 = 0) where {T <: InferenceVariable}  =
+    DFGVariable(label, now(), Symbol[],
+              Dict{Symbol, MeanMaxPPE}(),
+              Dict{Symbol, VariableNodeData{T}}(:default => VariableNodeData{T}()),
+              Dict{String, String}(),
+              Dict{Symbol,AbstractBigDataEntry}(), 0, _internalId)
+
+"""
+    $(SIGNATURES)
+Structure for first-class citizens of a DFGVariable.
+"""
+mutable struct DFGVariableSummary <: AbstractDFGVariable
+    label::Symbol
+    timestamp::DateTime
+    tags::Vector{Symbol}
+    estimateDict::Dict{Symbol, <:AbstractPointParametricEst}
```

Comment:
latest v0.5.3 has this as `ppeDict`

Reply:
see here: https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/blob/bea526695ad7f55ab695fd80b34dd74e3bcd60d6/src/entities/DFGVariable.jl#L186

Reply:
will likely fall out naturally when merging all this to master, just ask to keep an eye on it when we do please.